packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py

```diff
@@ -143,20 +143,13 @@ def wrapped_on_end(span):
             # this makes sure otel context is propagated so we always want it
             ThreadingInstrumentor().instrument()
 
-            instrument_set = init_instrumentations(
+            init_instrumentations(
                 should_enrich_metrics,
                 image_uploader.aupload_base64_image,
                 instruments,
                 block_instruments,
             )
 
-            if not instrument_set:
-                print(
-                    Fore.RED + "Warning: No valid instruments set. Remove 'instrument' "
-                    "argument to use all instruments, or set a valid instrument."
-                )
-                print(Fore.RESET)
```

Comment:
Removed as duplicate of the following:
https://github.com/traceloop/openllmetry/blob/78aa8a7fa611fb06e793032d374eda420a4a9a8c/packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py#L478-L485